test/e2e/testing-manifests/statefulset/multivolumezk/statefulset.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: multivolumezk
+spec:
+  serviceName: "multivolumezk"
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: multivolumezk
+      annotations:
+        pod.alpha.kubernetes.io/init-containers: '[
```

Comment:
Looks like they all need to be updated, which is a bigger PR.  Opened https://github.com/kubernetes/kubernetes/issues/45202